etc/base.yaml

```diff
@@ -90,18 +99,25 @@ confluent_cloud:
 
 cp_zookeeper:
   _install: true
-  _chart_version: 0.2.0
+  _chart_version: 0.3.1
+  imageTag: 7.2.0
   servers: 3
 
 cp_kafka:
   _install: true
-  _chart_version: 0.2.0
+  _chart_version: 0.3.0
+  imageTag: 7.2.0
+  customEnv:
+    # Set this to 3.2 when upgraded to 7.2.0
+    KAFKA_INTER_BROKER_PROTOCOL_VERSION: "1.1"
```

Comment:
For a fresh 7.2.0 install this should be 3.2?

Reply:
Yes, indeed

Reply:
Should we assume a fresh install like we do for other parts of the configuration?